### PR TITLE
[MIG] mrp_analytic: Migration to 11.0

### PR DESCRIPTION
**Pull request by SalahAdDin**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
It seems you haven't used properly the migration method, as there are other modules included in this PR: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

**Comment by pedrobaeza** [comment]
Using the method exposed, you don't need to remove the other addons, and it's the proper way to do it. Please try.

**Comment by SalahAdDin** [comment]
I tried, it didn't merge, i get many errors.
```
git am --continue
Aplicando: Move my module account_analytic_required from extra-addons to account-analytic
Sin cambios - olvidaste usar 'git add'?
Si no hay nada en el área de stage, las posibilidad es que algo mas
ya haya introducido el mismo cambio; tal vez quieras omitir este parche.
Cuando haya resuelto este problema, ejecute "git am --continue".
Si prefieres saltar este parche, ejecuta "git am --skip".
Para restaurar la rama original y detener el parchado, ejecutar "git am --abort".
```

**Comment by pedrobaeza** [comment]
You have to try to resolve conflicts.

**Comment by SalahAdDin** [comment]
Must i to resend this pull request?

**Comment by pedrobaeza** [comment]
You can use locally the same branch name and forced push your new commits without problems.

**Comment by SalahAdDin** [comment]
Done.

**Comment by pedrobaeza** [comment]
Great, you did it! Now I let space for the reviewers.

**Comment by SalahAdDin** [comment]
Thank you.

**Comment by SalahAdDin** [comment]
Hi, i'm testing that module, i'd created many manufacturing orders with respective analytic accounts but i am not getting the correct productions' number:
deepin-screen-recorder_select area_20180904115412

Why?

**Comment by SalahAdDin** [comment]
:up:  :+1:

**Comment by SalahAdDin** [comment]
I cannot understand what is the problem with the `flake8`'s test.

**Review by AaronHForgeFlow — COMMENTED** [review]
======== Testing test_flake8 ========
mrp_analytic/models/analytic_account.py:4:1: F401 'odoo.api' imported but unused
mrp_analytic/models/analytic_account.py:12:80: E501 line too long (88 > 79 characters)

**Comment by SalahAdDin** [comment]
The final error, i cannot understand it.

File: mrp_analytic/models/analytic_account.py
Comment (espo-tony) [inline mrp_analytic/models/analytic_account.py]: Travis says this line is too long. Find a way to write it within 80 characters, either splitting it on multiple lines or (best way, IMHO) renaming `analytic_account` into something shorter (`an_acc` or `record` should make the job)

File: mrp_analytic/README.rst
Comment (espo-tony) [inline mrp_analytic/README.rst]: Please, enter 11.0 runbot, not 8.0

**Review by espo-tony — CHANGES_REQUESTED** [review]

**Comment by SalahAdDin** [comment]
@espo-tony Ok, i did it, but i am still getting an error.

File: mrp_analytic/models/analytic_account.py
Comment (espo-tony) [inline mrp_analytic/models/analytic_account.py]: Please, put the 2 imports above on the same line

Comment (espo-tony) [reply mrp_analytic/models/analytic_account.py]: @SalahAdDin Thanks. Travis is not failing anymore. Runbot failing is something different, don't know exactly what's wrong with it but I suspect it doesn't directly depends from your PR.

Comment (SalahAdDin) [reply mrp_analytic/models/analytic_account.py]: > Please, put the 2 imports above on the same line

Must i to do this?

**Comment by SalahAdDin** [comment]
@espo-tony @aheficent Thank you very much for your reviews, in hour instance we need this module very much.

Thank you.

**Review by AaronHForgeFlow — APPROVED** [review]
@SalahAdDin Thanks to you!. Testing on runbot and working fine. :+1:

File: mrp_analytic/models/analytic_account.py
Comment (espo-tony) [reply mrp_analytic/models/analytic_account.py]: Yeah, it doesn't really change anything, but it's required to be compliant with OCA guidelines.

`from odoo import fields, models`

**Comment by AaronHForgeFlow** [comment]
a nice module also would be to pass the analytic accounts  to the stock moves when the stock_analytic module is merged

**Comment by SalahAdDin** [comment]
@aheficent I would take more time to add that feature, we can put it as a TODO task, and i can try it after, because right now i don't know how can i do that XD.

**Review by espo-tony — APPROVED** [review]
@SalahAdDin , Thank you for your work. This looks good to me :+1:

**Comment by AaronHForgeFlow** [comment]
@SalahAdDin Sure. It is just an idea, I will not work on that now but maybe in the following weeks. Stock_analytic is still not in 11.0 anyway

**Comment by SalahAdDin** [comment]
Thank you very much, just merging left.

**Comment by AaronHForgeFlow** [comment]
Can you squash your commits into one before?

**Comment by SalahAdDin** [comment]
@aheficent Last commits?

**Comment by AaronHForgeFlow** [comment]
The ones of this migration. From `[MIG] mrp_analytic: Migration to 11.0` onwards. I think that having just one with `[MIG] mrp_analytic: Migration to 11.0` is ok.

**Comment by SalahAdDin** [comment]
Done :D

**Comment by AaronHForgeFlow** [comment]
Waiting for CI :)

**Comment by SalahAdDin** [comment]
@moylop260 Hi man, i'm sorry but i need your help.
We have a problem here with runbot.

**Comment by SalahAdDin** [comment]
The other module is waiting for this module: https://github.com/OCA/manufacture/pull/296#issuecomment-428913061

**Pull request merged**
